backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/rabbitmq/RabbitMQAuditEntry.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.rabbitmq;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import org.adempiere.util.net.IHostIdentifier;
+
+import javax.annotation.Nullable;
+
+@Value
+@Builder
+public class RabbitMQAuditEntry
+{
+	@NonNull String queueName;
+	@NonNull RabbitMQDirection direction;
+	@NonNull Object content;
+	@Nullable String eventUUID;
+	@Nullable String relatedEventUUID;
+	@Nullable IHostIdentifier host;
```

Comment:
i think it would make sense to also have a property for indicating the service on which the "in" or "out" happened (app or webapi).
Then we could also use this for intra-metasfresh-auditing.

Reply:
also, we might add a "status", like success, warning or error..and in the case of error we might have an AD_Issue_ID